server/api/users/posts/[id].put.ts

```diff
@@ -0,0 +1,81 @@
+import { Posts, Users } from "../../../dbModels";
+import middlewareFunction from "../../../utils/middlewareFunction";
+
+interface IRequestBody {
+	channelData: string;
+}
+
+export default defineEventHandler(async (e) => {
+	//IF THE USER HAS EDIT OR ALL PERMISSIONS TO THIS CHANNEL THEN THE USER CAN EDIT ITS POSTS
+	const postId = e.context.params.id;
+	console.log(`PUT /api/user/posts/updatePost/${postId}`);
+	try {
+		const userId = await middlewareFunction(e);

```

Comment:
if the jwt token is invalid then the middleware will return false,where are we handling this?

Reply:
Yeah. I missed to add that code in put and delete operations of the user. I have fixed it. 
```
const userId = await middlewareFunction(e);
		console.log(userId);

		if (e.req.headers.authentication == null) {
			e.res.statusCode = 401;
			return { msg: "No token, authorization denied" };
		}

		if (userId == false) {
			e.res.statusCode = 401;
			return { msg: "Token is not valid" };
		}
```